packages/common/driver-definitions/src/storage.ts

```diff
@@ -509,4 +509,17 @@ export interface ISnapshotFetchOptions {
 	 * container, so specifying version is not necessary for storage services.
 	 */
 	versionId?: string;
+
+	/**
+	 * List of groupId of datastores for which the snapshot needs to be fetched. If not provided, entire snapshot
+	 * will be requested from the service. It is upto the service, to include snapshot for datastores with groupIds
+	 * or not.
+	 */
+	groupIds?: string[];
+
+	/**
+	 * Specify if you want default behavior of the driver to fetch the snapshot like lets say simultaneously fetch from
+	 * network and cache or specify FetchSource.noCache to just fetch from network.
+	 */
+	fetchSource?: FetchSource;
```

Comment:
Maybe browser / github fails me, but I do not see any place where it's used (passed in). What is the intention here? For Loader layer to control the policy? I'd love to understand more, including API usage changes (as Host would now need to communicate with Loader RE policy).
I think the intention here is for it to be overwrite for loading when groupIds is specified. I.e. default snapshot loading would use host policy (whatever it is), but when we need to load additional data stores (by gtoupID), loader will ask driver to not use cache (or something like that).
If that's the case, maybe it's better for such decisions to be implicit (derived from presence of groupIds)? Not sure.
Let me know, as it's not very clear on what is the design / flow.

Reply:
This option I just trasnferred from the getVersions api and it is used in containerRuntime.fetchLatestSnapshotFromStorage()

It is not related to Data virtualization. It was added originally, when the fetcher wants to make sure that the snapshot is not supplied from cache but is fetched from network.